lib/puppet/type/group.rb

```diff
@@ -96,7 +96,7 @@ def change_to_s(currentvalue, newvalue)
 
       def insync?(current)
         if provider.respond_to?(:members_insync?)
-          return provider.members_insync?(current, @should)
+          return provider.members_insync?(current, munge_members_value(@should))
```

Comment:
not sure about this modification, as generic behavior gets changes, could the change be done in libuser.rb?

Reply:
yes, I can implement this in `libuser#members_insync?`